packages/grpc-js/src/resolver-dns.ts

```diff
@@ -45,10 +44,8 @@ function trace(text: string): void {
  */
 export const DEFAULT_PORT = 443;
 
-const DEFAULT_MIN_TIME_BETWEEN_RESOLUTIONS_MS = 30_000;
-
-const resolveTxtPromise = util.promisify(dns.resolveTxt);
-const dnsLookupPromise = util.promisify(dns.lookup);
+const DEFAULT_MIN_TIME_BETWEEN_RESOLUTIONS_MS = 30_000,
+  DNS_RESOLUTION_ENV = 'GRPC_DNS_RESOLVER_TODO';
```

Comment:
Please handle the environment variable in `environment.ts`, following the examples in that file.